src/Aop/Framework/TraitIntroductionInfo.php

```diff
@@ -19,48 +19,48 @@ class TraitIntroductionInfo implements IntroductionInfo
 {
 
     /**
-     * List of interfaces to introduce
+     * Introduced interface
      *
-     * @var array
+     * @var string
      */
-    private $introducedInterfaces;
+    private $introducedInterface = '';
 
     /**
-     * List of traits to include
+     * Trait to use
      *
-     * @var array
+     * @var string
      */
-    private $introducedTraits;
+    private $introducedTrait = '';
```

Comment:
It's preparation for merging into master branch where strict type will be applied to the source code. So, return type hint will be `string` and we should not check if it's null or not.